### DISCUSSION TITLE
这和 Electron Forge 提供的 Vite 解决方案有什么区别吗？

### DISCUSSION THREAD

**Discussion by Mesimpler** [alex8088/electron-vite#278] [Q&A] [answered]
[electronforge vite templates](https://www.electronforge.io/templates/vite)
[electronforge vite plugin](https://www.electronforge.io/config/plugins/vite)

**Comment by alex8088** [comment, marked as answer]
electronforge vite 是一个基础的方案，而electron-vite 提供完善的解决方案，例如 预配置减少开发者的配置心智负担；开箱支持 Vue、React、Svelte 和 SolidJS 等；优化electron主进程资源处理；代码保护；易于调试；完善的开发文档等等，旨在为 Electron 提供更快、更精简的开发体验。

**Comment by rtritto** [comment]
@alex8088 for both, what are pro/cons and differences (features, performances etc)?
